pkg/services/images.go

```diff
@@ -130,6 +130,17 @@ func (s *ImageService) CreateImage(image *models.Image, account string, orgID st
 	if image.Version == 0 {
 		image.Version = 1
 	}
+	packages := image.Packages
+	// we now need to loop this request for each package
+	for _, p := range packages {
+		count, er := validateImagePackage(p.Name, image)
+		if er != nil {
+			return er
+		}
+		if (*count).Meta.Count != 1 {
```

Comment:
Think the right condition is 
```suggestion
		if (*count).Meta.Count == 0 {
```